### PR TITLE
Add utility pallet to parachain

### PR DESCRIPTION
**Pull request by sayyed-kamran**
Please review the changes to add utility pallet to the parachain

### CODE REVIEW

File: runtime/peaq-dev/src/lib.rs [hunk 1]
```diff
@@ -783,6 +791,7 @@ construct_runtime!(
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 5,
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 6,
 		Contracts: pallet_contracts::{Pallet, Call, Storage, Event<T>} = 7,		
+		Utility: pallet_utility::{Pallet,Call,Event}=8,
```
Comment (sfffaaa) [inline runtime/peaq-dev/src/lib.rs]: Please modify the format.

File: runtime/peaq-dev/src/lib.rs [hunk 2]
```diff
@@ -447,6 +447,14 @@ impl peaq_pallet_did::Config for Runtime {
     type WeightInfo = peaq_pallet_did::weights::SubstrateWeight<Runtime>;
 }
 
+/// Config the utility in pallets/utility
+impl pallet_utility::Config for Runtime {
+	type Call = Call;
+	type Event = Event;
+	type PalletsOrigin = OriginCaller;
+	type WeightInfo = ();
+	}
```
Comment (sfffaaa) [inline runtime/peaq-dev/src/lib.rs]: Please change the indent.

File: runtime/peaq-dev/src/lib.rs (hunk 1 above)
Comment (sayyed-kamran) [reply runtime/peaq-dev/src/lib.rs]: Formatted

File: runtime/peaq-dev/src/lib.rs (hunk 2 above)
Comment (sayyed-kamran) [reply runtime/peaq-dev/src/lib.rs]: Indent set

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**